bootstrap.html

```diff
@@ -9,36 +9,48 @@
   <link rel="stylesheet" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.14/themes/base/jquery-ui.css" type="text/css" media="all">
   
   
+  <style>
+    table{
+        margin:60px;
+        border-collapse: collapse;
+        border-spacing: 0;
+    }
+    th, td{
+        padding: 10px 20px;
+        border: 1px solid #000;
+    }
+</style>
+  
 </head>
-<body>
+<body id="body">

```

Comment:
You don't need to create id for body, there is no other body tag in normal cases

Reply:
i give it just to call it by id in javascript so i can append the created html element

Reply:
i give it just to call it by id in javascript so i can append the created html element

Reply:
I understand it, but you can use getElementsByTagName